rest-api-guidelines/core-principles/design-platform.md

```diff
@@ -0,0 +1,12 @@
+# API Design Platform
+
+1. [SwaggerHub](https://swageri.io/) is the primary platform supporting [API first approach](../../general-guidelines/api-first.md). SwaggerHub **SHOULD** be used during API Design.
+2. Every API description **MUST** be stored in [SwaggerHub](https://http://design.api.3stripes.io/) under the adidas team.
+3. SwaggerHub **MUST** be the **single source of truth** to learn about existing APIs within the organization.
+
+> NOTE: SwaggerHub supports API-first approach in multiple ways:  
+> For example, it validates API description for correctness and automatically generates API documentation to drive the discussion between stakeholders. \(No more emails with API description flying between stakeholders\)
```

Comment:
for correctness -> for the correctness

Reply:
I think it's correct actually. _For the correctness_ would be **_of_** something. In this case it is absolutely fine :) 
Swaggerhub as a Catalyst of engagement. I love this over the top terminology ;) We could put it here to sound more grandiose :D